### PR TITLE
Blog - whats new 4.9. WIP

### PR DESCRIPTION
**Pull request by davsclaus**

### CODE REVIEW

**Review by squakez — COMMENTED** [review]
I'm going to include a short paragraph about the usage of camel-observability-services.

**Comment by davsclaus** [comment]
Yes anyone is welcome to help and contribute to this PR

File: content/blog/2024/12/camel49-whatsnew/index.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+You can now configure kamelets with ENV variables using a more human ready for lang parameters.
+
+The option `bucketNameOrArn` can now be configured using both of the following styles:
+
+```properties
+CAMEL_KAMELET_AWS_S3_SOURCE_BUCKETNAMEORARN = myBucket
+CAMEL_KAMELET_AWS_S3_SOURCE_BUCKET_NAME_OR_ARN = myBucket
+```
+
+This actually applies to all the Camel configuration you can (not only for kamelets).
```
Comment (squakez) [inline content/blog/2024/12/camel49-whatsnew/index.md:112]:
```suggestion
This actually applies to all the Camel configuration you can (not only for Kamelets).
```

**Review by squakez — APPROVED** [review]

**Comment by davsclaus** [comment]
@tdiesler you are welcome to help with some docs on camel-jbang kubernetes plugin and what you have done for the 4.9 release.

**Pull request merged**